plugin/storage/es/spanstore/writer.go

```diff
@@ -140,12 +141,15 @@ func (s *SpanWriter) createIndex(indexName string, mapping string, jsonSpan *jMo
 		exists, _ := s.client.IndexExists(indexName).Do(s.ctx) // don't need to check the error because the exists variable will be false anyway if there is an error
 		if !exists {
 			// if there are multiple collectors writing to the same elasticsearch host, if the collectors pass
```

Comment:
this whole sentence is kinda wonky now.
"if there are multiple collectors attempting to create the same index at once, we make sure the error is not index_already_exists_exception or else we return the error"

or something to that effect